src/transformers/models/pegasus/tokenization_pegasus.py

```diff
@@ -206,10 +218,6 @@ def _special_token_mask(self, seq):
         all_special_ids = set(self.all_special_ids)  # call it once instead of inside list comp
         all_special_ids.remove(self.unk_token_id)  # <unk> is only sometimes special
 
-        assert all_special_ids == set(
```

Comment:
This assert was pretty bad IMO -> It essentially forced the first 4 tokens to be pad_token, eos_token, unk_token, mask_token, so one wasn't even allowed to set `mask_token_id` to a value > 3 => delete the assert

Reply:
@LysandreJik @sgugger

Reply:
Works for me